src/main/java/org/jabref/logic/xmp/DublinCoreExtractor.java

```diff
@@ -61,12 +63,12 @@ private void extractAuthor() {
     }
 
     /**
-     * Year + Month in BibTex - Date in DublinCore is a combination of year and month information
+     * Year in BibTex - Date in DublinCore is only the year information, because dc interprets empty months as January.
```

Comment:
I don't get this change? The old code from line 81 on seems to extract the month perfectly. In case no month is there, the fallback was and still seems to be January.